src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+import * as FaIcons from 'react-icons/fa'
+import * as AiIcons from 'react-icons/ai'
+import PropTypes from 'prop-types'
+
+import SearchBar from '../SearchBar/SearchBar.component'
+import ToggleSwitch from '../ToggleSwitch'
+
+import Colors from '../../styles/Colors'
+import Button from '../Button'
+
+function Header(props) {
+    const [isToggled, setIsToggled] = useState(false)
+    const pressHandler = () => props.func()
```

Comment:
**No big deal**:

A better name for `.func()` could be `.onPressHandler()` and an even better could be `onMenuIconClick` or `onHamburgerMenuClick()` so that it is more descriptive of the actual function that it is doing and it is easier to know at a glance wherever you use the header component like in `Sidebar`

Reply:
You're right, I forgot to give it a nice and descriptive name, I'm going to use 'onHamburgerMenuClick' :)

Reply:
https://github.com/G-Dyz/react-certification-2021/blob/c46af309f75c1e2d0259b18d4bcb503afd6847f2/src/components/Sidebar/Sidebar.component.jsx#L50